### ISSUE TITLE
A11y: The Menu is a list but not marked as a list (ul, li)

### ISSUE THREAD

**Issue by daniellekirkwood** [vector-im/element-web#24928]
Using the appropriate HTML structural elements ensures that the structure of a
page is fixed and accessible regardless of the presentation.

This is also present on other submenus.

<img width="353" alt="Screenshot 2023-03-24 at 12 46 17" src="https://user-images.githubusercontent.com/89144281/227524949-2d196372-ae25-4836-b016-a58991cb75da.png">

**Comment by t3chguy** [comment]
The WAI ARIA example here explicitly nullifies the role on `li` elements - https://www.w3.org/WAI/ARIA/apg/patterns/menu-button/examples/menu-button-links/ - implying this request is invalid

**Comment by daniellekirkwood** [comment]
Great, let's close it out :)

**Comment by daniellekirkwood** [comment]
![Image](https://user-images.githubusercontent.com/89144281/232091047-ef94dda8-0456-4b50-8633-9a2b38ff200c.png)



![Image](https://user-images.githubusercontent.com/89144281/232091146-910f65b5-253a-44ee-a6ed-c55f2178096a.png)


@t3chguy can you confirm that your above comment applies to these two areas also?

**Comment by t3chguy** [comment]
@daniellekirkwood things can only be of one role, i.e. list. The tab controls in settings should be just that, tab controls. The pop up menu should be of role menu, not list.

**Comment by daniellekirkwood** [comment]
Great - thanks :) 

In the other reports, these were raised as issues similarly to the OG in this GH... Just wanted to check the same applies here so i can respond

**Comment by Morhonda** [comment]
Hi @daniellekirkwood and @t3chguy,

has this been merged and in which version of Element is this fixed? thanks.

Best wishes,

Moritz

**Comment by t3chguy** [comment]
@Morhonda I suggest reading the previous responses, especially https://github.com/vector-im/element-web/issues/24928#issuecomment-1482746382
